scripts/load/load_redistimeseries.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+EXE_FILE_NAME=./bin/tsbs_load_redistimeseries
+
+set -x
+set -e
+
+EXE_DIR=${EXE_DIR:-$(dirname $0)}
+source ${EXE_DIR}/../redistimeseries_common.sh
+
+# Ensure RESULTS DIR available
+mkdir -p ${RESULTS_DIR}
+
+OUT_FULL_FILE_NAME="${RESULTS_DIR}/${PREFIX}_load_result.out"
+echo "Saving results to ${OUT_FULL_FILE_NAME}"
+
+# Load new data
+$EXE_FILE_NAME \
+  --file ${DATA_FILE_NAME} \
+  --workers=${NUM_WORKERS} \
+  --batch-size=${BATCH_SIZE} \
+  --reporting-period=${REPORTING_PERIOD} \
+  --host=${DATABASE_HOST}:${DATABASE_PORT} ${CLUSTER_FLAG} \
+  --connections=${CONNECTIONS} --pipeline=${PIPELINE} |
```

Comment:
I'm missing the `--results-file` option here. Any reason to skip it?

Reply:
I believe it was due to the fact when I started working on this branch we did not had that option implemented/merged. Will do the change

Reply:
@jonatas fixed in the latest commits